README.md

```diff
@@ -45,6 +45,37 @@ aedesPersistenceRedis({
 })
 ```
 
+Alternatively, you can pass in an externally created Redis connection using the
+`conn` option. This can be useful when connecting to a Redis cluster, for example.
+
+Example:
+
+```js
+aedesPersistenceRedis({
+  conn: new Redis.Cluster([{
+    port: 6379,
+    host: '127.0.0.1'
+  }, {
+    port: 6380,
+    host: '127.0.0.1'
+  }])
+})
+```
+
+The same as above but with `clusters` option:
+
+```js
+aedesPersistenceRedis({
+  clusters: [{
```

Comment:
We could call this `cluster` instead of `clusters`, since we are connecting to a single cluster.

Reply:
You right :)